crates/turbopack-ecmascript/src/chunk_group_files_asset.rs

```diff
@@ -119,16 +119,23 @@ impl EcmascriptChunkItem for ChunkGroupFilesChunkItem {
     #[turbo_tasks::function]
     async fn content(&self) -> Result<EcmascriptChunkItemContentVc> {
         let chunks = self.inner.chunks();
-        let mut data = Vec::new();
         let base_path = self.inner.await?.base_path.await?;
-        for chunk in chunks.await?.iter() {
-            let path = chunk.path().await?;
-            if let Some(p) = base_path.get_path_to(&path) {
-                data.push(Value::String(p.to_string()));
-            }
-        }
+        let chunks_paths = chunks
+            .await?
+            .iter()
+            .map(|chunk| chunk.path())
+            .try_join()
+            .await?;
+        let chunks_paths: Vec<_> = chunks_paths
+            .iter()
+            .filter_map(|path| base_path.get_path_to(&path))
+            .collect();
         Ok(EcmascriptChunkItemContent {
-            inner_code: format!("__turbopack_export_value__({:#});\n", Value::Array(data)).into(),
+            inner_code: format!(
+                "__turbopack_export_value__({});\n",
+                stringify_js_pretty(&chunks_paths)
```

Comment:
It would be cool to avoid the temp string here by having a `Stringify<T>` struct that implements `Display` and `Debug`

Reply:
Created WEB-688 to track this.